third_party/move/move-model/bytecode/src/stackless_control_flow_graph.rs

```diff
@@ -251,6 +299,74 @@ impl StacklessControlFlowGraph {
     }
 }
 
+/// Iterator over blocks of a control flow graph in DFS order
+pub struct DFSLeft<'a> {
+    cfg: &'a StacklessControlFlowGraph,
+    // blocks scheduled to visit
+    to_visit: Vec<BlockId>,
+    // blocks visited
+    visited: BTreeSet<BlockId>,
+    // blocks unvisited, used only when iterating over all blocks
+    unvisited: Option<BTreeSet<BlockId>>,
+}
+
+impl<'a> DFSLeft<'a> {
+    /// Iterates over all blocks in DFS order (and always choosing the left-most child to visit) starting from the entry block
```

Comment:
Oh, this "left-most child" aspect should be in the earlier comments.  (Including the struct comment.)

Reply:
?

Reply:
done